src/Abp.RedisCache/Runtime/Caching/Redis/DefaultRedisCacheSerializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Abp.Json;
+using StackExchange.Redis;
+
+namespace Abp.Runtime.Caching.Redis
+{
+    /// <summary>
+    ///     Default implementation uses JSON as the underlying persistence mechanism.
+    /// </summary>
+    public class DefaultRedisCacheSerializer : IRedisCacheSerialization
```

Comment:
I named it IRedisCacheSerialization as IRedisCacheSerializer.Serialize() looked weird as Serializer,Serialize(). Will however update naming and let you have new PR.